packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -81,6 +81,29 @@ describe('common/sqlite', () => {
     expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
+  test('basic introspection with schema and --url missing file: prefix should fail', async () => {
```

Comment:
Previously the error was P1012, see https://github.com/prisma/prisma/issues/14732